api/src/main/java/io/opentelemetry/correlationcontext/DefaultCorrelationContextManager.java

```diff
@@ -50,7 +50,7 @@ public static CorrelationContextManager getInstance() {
 
   @Override
   public CorrelationContext getCurrentContext() {
-    return ContextUtils.getValue();
+    return CorrelationsContextUtils.getCorrelationContext(Context.current());
   }
```

Comment:
Can we file an issue to remove these? Also I would suggest removing them because adding them later is backwards  compatible.

Reply:
I guess this is what you are talking about? ;) https://github.com/open-telemetry/opentelemetry-specification/issues/455

Reply:
Is this solved? Or this is something else you are talking about?